portfolio/src/main/webapp/script.js

```diff
@@ -27,5 +28,13 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
+function googleText(id) {
```

Comment:
You don't seem to be using the `id` argument.
In fact, if you did pass in `google` and `pru` for the other one, you can just reduce code duplication and just use one `toggleText(divId)` function for both, where you use the argument as in `document.getElementById(divId);`.